downloads.md

```diff
@@ -168,6 +168,7 @@ All Flink releases are available via [https://archive.apache.org/dist/flink/](ht
 
 ### Flink
 
+- Flink 1.7.2 - 2019-02-12 ([Source](https://archive.apache.org/dist/flink/flink-1.7.2/flink-1.7.2-src.tgz), [Binaries](https://archive.apache.org/dist/flink/flink-1.7.2/), [Docs]({{site.DOCS_BASE_URL}}flink-docs-release-1.7/), [Javadocs]({{site.DOCS_BASE_URL}}flink-docs-release-1.7/api/java), [ScalaDocs]({{site.DOCS_BASE_URL}}flink-docs-release-1.7/api/scala/index.html))
```

Comment:
Hi @tzulitai , the `downloads.zh.md` file is created and not translated yet. So it would be nice to update `downloads.zh.md` too.

Thanks.

Reply:
Thanks, I'll update `downloads.zh.md`